CHANGELOG.md

```diff
@@ -11,6 +11,84 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - automatic range measuring when using targeting
   ([#75](https://github.com/Wasteland-Ventures-Group/WV-VTT-module/issues/75))
+- weapons
+  - Alien Blaster
+  - Arcane Glove
+  - Auto-Axe
+  - Balefire Egg Launcher
+  - Baseball Bat
+  - Bladed Gauntlets
+  - Buckington Automatic Rifle M18
+  - Buckscatter G-M7 Launcher
+  - Dynamite
+  - Enclave Gale Blaster
+  - Enclave Hailgun
+  - Enclave Novasurge Rifle
+  - Enclave Rainmaker
+  - Enclave Thunderbuck
+  - Frag Grenade
+  - GAG Incinerator
+  - GAG Thunder Cannon
+  - Gauss Minigun
+  - Gauss Pistol
+  - Gauss Revolver
+  - Gauss Rifle
+  - Gauss Shotgun
+  - Gauss SMG
+  - Gretta M28 MG "Pecker"
+  - Gretta M33 MG "Predator"
+  - Gretta M3 SMG "Greaser"
+  - H&K MGat-39 "Mini Gun"
+  - Hellhound Gauntlets
+  - Hoof of Goddess
+  - Hunting Bow
+  - IF-44 "Angel Bunny"
+  - IF-84 "Stampede"
+  - IF-92 "Melrose"
+  - IF-G12 "Champion"
+  - Improvised Melee (small)
+  - Longbow
+  - Lumber Axe
+  - Mace
+    - Manticore Gauntlets
```

Comment:
The indent here is wrong. It should be one less.

Reply:
Removed extra indent